### ISSUE TITLE
Feature request: display player stats.json on profile page

### ISSUE THREAD

**Issue by redbau** [NamelessMC/Nameless#1441]
TLDR: Display Minecraft statistics from the uuid.json file on player's forum profile pages:
```
Played 52 days on purple survival server
Mined 1.2K diamonds on purple survival server
Walked 12M blocks on red survival server
...
Placed 132K blocks on creative server
```

The best thing about Nameless is it acts as a bridge between my Minecraft server and all the awesome player activity going on there, and my website, where anybody can see the progress of the server and get a peek into this activity without having to login to the game.

So Minecraft already has built-in statistics for every player. I'd love to have access to that data from the website to do things like show how many diamonds a player has mined on X survival server, or how many blocks placed on Y creative server. 

For example, I currently pull this data into my own MySQL table and display it on a ladder page: https://wildercraft.net/ladder/ — this is on a separate page as part of my Nameless site, but wouldn't this be nice to include everywhere because it's synced with Nameless?

**Comment by samerton** [comment]
Going to link this with #1484, thinking this can be one stats module that supports many different input methods.

**Comment by Derkades** [comment]
PlaceholderAPI does have placeholders for these statistics (I think?) so when #1484 is implemented, this will be possible too.

**Comment by Mitzythecat** [comment]
I think maybe you can integrate this with the stats plugin named plan - it has a very good stat system. Maybe this could be a bungee plugin that hooks into plans api? That would be cool

**Comment by redbau** [comment]
Thanks for the update @Derkades!

**Comment by Derkades** [comment]
Just to clarify, I did not implement this feature but closed the issue because the same thing can be done using #1484
